### PR TITLE
fix amp-bind sample

### LINKED ISSUE #1816: amp-bind [class]
When using amp-bind to bind the [class] attribute, the initialization page reports an error: %s: "%s" is not a valid result for [class]. amp-bind null. What should I do with this problem? I also saw an error in the template page. You can see the console at this address is reporting an error: https://ampbyexample.com/components/amp-bind/source/#development=1

### PR DESCRIPTION
**Pull request by kul3r4**
Fixes #1816

### CODE REVIEW

File: src/20_Components/amp-bind.html [hunk 1]
```diff
@@ -65,6 +65,7 @@
       You can dynamically change the CSS classes of an element by adding a binding to the `[class]` attribute.
     -->
     <div>
+      <amp-state id="myClass"><script type="application/json">""</script></amp-state>
```
Comment (sebastianbenz) [inline src/20_Components/amp-bind.html]: We need to restructure the sample, as we haven't introduced the amp-state element at this point.

Comment (kul3r4) [reply src/20_Components/amp-bind.html]: Should I move this section after the part that introduces `amp-state`?

Comment (sebastianbenz) [reply src/20_Components/amp-bind.html]: How about moving the initialising state sample after binding text?

Comment (kul3r4) [reply src/20_Components/amp-bind.html]: done

**Comment by gaganabhay** [comment]
@sebastianbenz  Can you please merge it. Need this merge very badly. Thanks

**Pull request merged**